### ISSUE TITLE
Just use issues instead of RFC process

### ISSUE THREAD

**Issue by hamza1311** [rustwasm/gloo#136]
The RFC process is complicated for how small gloo is. It's an informal version of RFC process used by Rust(/Wasm) but that stills make it overkill(?) for gloo. It also pushes away contributors (it at least pushed me away initially) because of it's complexity. I suggest that we use "Feature proposal" type issues instead.

### Currently open RFCs

#129 
#117 
#79 

Those will be converted to issues, should this change take place.

**Comment by jstarry** [comment]
I feel the same way, this change would be welcome!

**Comment by hamza1311** [comment]
Let's go with it then. I've pinged the authors of RFCs to create the issues. If they don't/can't do it, I will create those issues and close the PRs. We also need to convert the accepted but unimplemented RFCs into issues

**Comment by DesmondWillowbrook** [comment]
Should we be copy-pasting the entire text of the RFC into the issue as well, since the Pull Request flow works on the basis of the RFC itself being version controlled and revisions are suggested through GitHub's review process?

**Comment by hamza1311** [comment]
I'd just copy/paste. If you think of a better way, I'd like to hear it. RFC process just doesn't work well with the small-scale of gloo

**Comment by mkroehnert** [comment]
Hi all, I have a short question related to moving the RFC process.

First off, I find it a good idea to simplify the process.
However, is there a plan to copy the finalized text from the PRs into a textfile inside the repository?
Reason behind the question: if you clone the repo you then get all relevant data also in one place. Also, if the repo may be moved to another service, it may not be guaranteed, that all isues and discussions can be migrated easily.

**Comment by hamza1311** [comment]
> However, is there a plan to copy the finalized text from the PRs into a textfile inside the repository?

No, once the feature/issue is implemented, it is closed via the PR that implements it. Repo only contains the code and docs with it. Tbh, gloo is small and we need to reduce the barrier to entry for contributions.

**Comment by DesmondWillowbrook** [comment]
Currently, the RFCs that were accepted are stored in the `rfcs` folder. Will we be removing those as well?
I rather like the idea of preserving the documents that dictated the design of the APIs within the repository itself, rather than external issues or PRs.

**Comment by hamza1311** [comment]
As I mentioned, gloo is small so dealing with documents and stuff is going to push away contributors. GitHub isn't going anywhere so I don't see the downside of not using the features it offers
